AdminWebsite/AdminWebsite.UnitTests/Controllers/PersonsControllerTest.cs

```diff
@@ -55,35 +56,183 @@ public void Setup()
         }
 
         [Test]
-        public async Task Should_return_request_if_match_to_search_term()
+        public async Task Should_return_searched_participants()
         {
-            _response.Add(new PersonResponse
-                            {
-                                Id = Guid.NewGuid(),
-                                ContactEmail = "jackman@hmcts.net1",
-                                FirstName = "Jack",
-                                LastName = "Mann",
-                                TelephoneNumber = "111222333",
-                                Title = "Mr",
-                                MiddleNames = "No",
-                                Username = "jackman@hmcts.net"
+            // Arrange
+            var additionalParticipantToReturn = new PersonResponse
+            {
+                Id = Guid.NewGuid(),
+                ContactEmail = "jackfilter@hmcts.net",
+                FirstName = "Filter",
+                LastName = "Participant",
+                TelephoneNumber = "111222333",
+                Title = "Mr",
+                MiddleNames = "No",
+                Username = "jackfilter@hmcts.net"
+            };
+            _response.Add(additionalParticipantToReturn);
+            
+            var expectedResponse = new List<PersonResponse>();
+            expectedResponse.Add(_response[0]);
+            expectedResponse.Add(additionalParticipantToReturn);
+            
+            _bookingsApiClient.Setup(x => x.PostPersonBySearchTermAsync(It.IsAny<SearchTermRequest>()))
```

Comment:
following on my comment below, so here instead of using It.IsAny<SearchTermRequest>() we use It.Is<SearchTermRequest>(request => request.Term == searchTerm) and then return the response